echopype/tests/visualize/test_plot.py

```diff
@@ -175,3 +177,142 @@ def test_plot_mvbs(
     mvbs = echopype.preprocess.compute_MVBS(Sv, ping_time_bin='1S')
     plot = echopype.visualize.create_echogram(mvbs)
     assert isinstance(plot, FacetGrid) is True
+
+
+@pytest.mark.parametrize(
+    ("water_level", "expect_warning"),
+    [
+        (True, False),
+        ([True], True),
+        (False, True),
+        (xr.DataArray(np.array(50.0)).expand_dims({'frequency': 3}), False),
+        (xr.DataArray(np.array(50.0)), False),
+        (10, False),
+        (30.5, False),
+    ],
+)
+def test_water_level_echodata(water_level, expect_warning):
+    from echopype.echodata import EchoData
+    from echopype.visualize.api import _add_water_level
+
+    filepath = ek60_path / "ncei-wcsd" / "Summer2017-D20170719-T211347.raw"
+    sonar_model = "EK60"
+    range_kwargs = {}
+
+    echodata = echopype.open_raw(
+        sonar_model=sonar_model, raw_file=filepath, xml_path=None
+    )
+
+    range_in_meter = echodata.compute_range(
+        env_params=range_kwargs.get('env_params', {}),
+        azfp_cal_type=range_kwargs.get('azfp_cal_type', None),
+        ek_waveform_mode=range_kwargs.get('waveform_mode', 'CW'),
+        ek_encode_mode=range_kwargs.get('encode_mode', 'power'),
+    )
+    single_array = range_in_meter.isel(frequency=0, ping_time=0).values
+    no_water_level = False
```

Comment:
suggest changing this to `no_input_water_level` to clarify what this does.

Reply:
```suggestion
    no_input_water_level = False
```